### PR TITLE
[TEST] disable PeakPickerWavelet test 3 (fails on travis with gcc)

### PR DESCRIPTION
**Pull request by hroest**
this should disable it on travis for gcc only

- see also PR #1555 
- see also #1202 #1205

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -9,9 +9,9 @@ matrix:
     - env: ENABLE_STYLE_TESTING=ON WITH_GUI=ON
       compiler: gcc
 ## split class vs. TOPP tests for GCC (otherwise it will run into time limit of 50min)
-    - env: ENABLE_STYLE_TESTING=OFF ENABLE_TOPP_TESTING=ON ENABLE_CLASS_TESTING=OFF WITH_GUI=ON
+    - env: ENABLE_STYLE_TESTING=OFF ENABLE_TOPP_TESTING=ON ENABLE_CLASS_TESTING=OFF WITH_GUI=ON DISABLE_PPWAVELET3=ON
```
Comment (cbielow) [inline .travis.yml]: adding the ENV var is not enough, you need to modify the travis sh file and pass an option as well.

**Comment by timosachsenberg** [comment]
I close this one as we have a temorary solution by running the build in debug mode

**Comment by hroest** [comment]
it seems that #1578 did not resolve the issue

**Comment by hroest** [comment]
travis should be a quick way to check whether a build succeeds. If we have a constantly failing test, it does not help anybody and actually slows down the whole process

we still need to figure out the root cause but to get travis working again as a tool for development, leaving out unreliable tests makes a lot of sense

File: tools/travis/lnx-cibuild.cmake [hunk 2]
```diff
@@ -15,13 +15,14 @@ message(STATUS "CTEST_BINARY_DIRECTORY: ${CTEST_BINARY_DIRECTORY}")
 set(INITIAL_CACHE
 "CMAKE_PREFIX_PATH=$ENV{SOURCE_DIRECTORY}/contrib\;/usr
 BOOST_USE_STATIC=Off
-CMAKE_BUILD_TYPE=Release
+CMAKE_BUILD_TYPE=$ENV{CMAKE_BUILD_TYPE}
```
Comment (hroest) [inline tools/travis/lnx-cibuild.cmake]: it seems that before, travis built in release mode anyways and ignored the variable introduced in #1578

**Comment by timosachsenberg** [comment]
+1

**Comment by hroest** [comment]
it seems travis just kills the linker when using debug mode http://cdash.openms.de/viewBuildError.php?buildid=143019 so maybe we are stuck with release mode

**Comment by hroest** [comment]
seems like we have a working test environment again

**Pull request merged**